common/app/views/fragments/metaData.scala.html

```diff
@@ -113,3 +113,11 @@
 }
 
 <meta name="google-site-verification" content="LR-FN6c2gIEUoo3k049w1nxyHykmac5ZE3SaUOiKc30" />
+
+@* Metadata to deep-link into native apps from facebook. http://applinks.org/documentation/ *@
+<meta property="al:ios:url" content="theguardian://">
```

Comment:
I assume this will just take the user to the apps homescreen? Surely we want to deep link straight o the current bit of content?